test/jdk/jdk/jfr/api/recording/event/TestShortPeriod.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
@parttimenerd Since you wrote the test, it should have SAP copyright.

Reply:
"Oracle" -> "SAP"?

Reply:
Yes, should look like here: https://github.com/openjdk/jdk/blob/master/test/jdk/jdk/nio/zipfs/TestPosix.java

Although for SAP we usually don't use a comma before SAP... so it should probably be:
`Copyright (c) 2022 SAP SE. All rights reserved.`